src/tokens/theme.web.js

```diff
@@ -93,67 +93,69 @@ const darkTheme = {
+      600: colors.xDarkBlue[600],
+      800: colors.xDarkBlue[700],
     },
     shade: {
-      800: colors.cloud[800],
-      900: colors.cloud[900],
-      910: colors.cloud[910],
-      920: colors.cloud[920],
-      930: colors.cloud[930],
-      940: colors.cloud[940],
-      950: colors.cloud[950],
-      960: colors.cloud[960],
-      970: colors.cloud[970],
-      980: colors.cloud[980],
+      500: colors.xCloud[500],
+      600: colors.xCloud[600],
+      700: colors.xCloud[700],
+      800: colors.xCloud[800],
+      900: colors.xCloud[900],
+      910: colors.xCloud[910],
+      920: colors.xCloud[920],
+      930: colors.xCloud[930],
+      940: colors.xCloud[940],
+      950: colors.xCloud[950],
+      960: colors.xCloud[960],
+      970: colors.xCloud[970],
+      980: colors.xCloud[980],
     },
     tone: {
-      800: colors.slate[800],
-      900: colors.slate[900],
-      920: colors.slate[920],
-      930: colors.slate[930],
-      940: colors.slate[940],
-      950: colors.slate[950],
-      960: colors.slate[960],
-      980: colors.slate[980],
+      900: colors.xSlate[900],
+      920: colors.xSlate[920],
+      930: colors.xSlate[930],
+      940: colors.xSlate[940],
+      950: colors.xSlate[950],
+      960: colors.xSlate[960],
+      980: colors.xSlate[980],
     },
     primary: {
-      100: colors.azzure[100],
-      200: colors.azzure[200],
-      300: colors.azzure[300],
-      400: colors.azzure[400],
-      500: colors.azzure[500],
-      600: colors.azzure[600],
-      700: colors.azzure[700],
-      800: colors.azzure[800],
-      900: colors.azzure[900],
-      910: colors.azzure[910],
-      920: colors.azzure[920],
-      930: colors.azzure[930],
-      940: colors.azzure[940],
-      950: colors.azzure[950],
-      970: colors.azzure[970],
+      100: colors.xAzzure[100],
+      200: colors.xAzzure[200],
+      300: colors.xAzzure[300],
+      400: colors.xAzzure[400],
+      500: colors.xAzzure[500],
+      600: colors.xAzzure[600],
+      700: colors.xAzzure[700],
+      800: colors.xAzzure[800],
+      900: colors.xAzzure[900],
+      910: colors.xAzzure[910],
+      920: colors.xAzzure[920],
+      930: colors.xAzzure[930],
+      940: colors.xAzzure[940],
+      950: colors.xAzzure[950],
+      970: colors.xAzzure[970],
     },
     negative: {
-      900: colors.rose[900],
-      930: colors.rose[930],
-      960: colors.rose[960],
+      900: colors.xRose[900],
+      930: colors.xRose[930],
+      960: colors.xRose[960],
     },
     positive: {
-      900: colors.green[900],
-      930: colors.green[930],
-      960: colors.green[960],
+      900: colors.xGreen[900],
+      930: colors.xGreen[930],
+      960: colors.xGreen[960],
     },
     neutral: {
-      900: colors.honey[900],
-      930: colors.honey[930],
-      960: colors.honey[960],
+      900: colors.xHoney[900],
+      930: colors.xHoney[930],
+      960: colors.xHoney[960],
     },
     light: {
```

Comment:
the 800 again seems out of sync with the theme.

Reply:
updated